.werft/vm/manifests.ts

```diff
@@ -0,0 +1,149 @@
+type NamespaceManifestOptions = {
+  namespace: string
+}
+
+export function NamespaceManifest({ namespace }: NamespaceManifestOptions) {
+  return `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${namespace}
+`
+}
+
+type VirtualMachineManifestArguments = {
+  vmName: string
+  namespace: string
+  claimName: string,
+  userDataSecretName: string
+}
+
+export function VirtualMachineManifest({ vmName, namespace, claimName, userDataSecretName }: VirtualMachineManifestArguments) {
+  return `
+apiVersion: kubevirt.io/v1
+type: kubevirt.io.virtualmachine
+kind: VirtualMachine
+metadata:
+  namespace: ${namespace}
+  annotations:
+    harvesterhci.io/volumeClaimTemplates: '[{"metadata":{"name":"${claimName}","annotations":{"harvesterhci.io/imageId":"default/image-cjlm2"}},"spec":{"accessModes":["ReadWriteMany"],"resources":{"requests":{"storage":"10Gi"}},"volumeMode":"Block","storageClassName":"longhorn-image-cjlm2"}}]'
+    network.harvesterhci.io/ips: "[]"
+  labels:
+    harvesterhci.io/creator: harvester
+    harvesterhci.io/os: ubuntu
+  name: ${vmName}
+spec:
+  running: true
+  template:
+    metadata:
+      annotations:
+        harvesterhci.io/sshNames: "[]"
+      labels:
+        harvesterhci.io/vmName: ${vmName}
+    spec:
+      domain:
+        hostname: ${vmName}
+        machine:
+          type: q35
+        cpu:
+          cores: 1
+          sockets: 1
+          threads: 1
+        devices:
+          inputs:
```

Comment:
Is this an input we want / need?

Reply:
It was part of the default configuration - let me try to remove it and see if things still work ☺️

Reply:
Serial Console still works even after removing the input devices ✅